### DISCUSSION TITLE
Realtek RTL8852AE Wifi 6 802.11ax PCIe Adapter

### DISCUSSION THREAD

**Discussion by Senmiko** [chris1111/WirelessAdapterCloverBigSur#13] [General]
Is it possible to make a kext for Realtek RTL8852AE Wifi 6 802.11ax PCIe Adapter so it works on macOS ventura at least?

**Comment by chris1111** [comment]
It would be great because this card looks incredible but the Source Code would have to exist

**Reply by Senmiko** [reply]
there is a source code for a kext but for linux
https://github.com/lwfinger/rtw89
I'm not sure if this can help
